### PR TITLE
oscillators: loop unrolling for wavetable oscillators

### PR DESCRIPTION
**Pull request by timblechmann**
gives a speedup of roughly 20%

### CODE REVIEW

File: server/plugins/OscUGens.cpp [hunk 1]
```diff
@@ -1414,17 +1414,45 @@ force_inline void Osc_ika_perform(OscType *unit, const float * table0, const flo
 
 	int32 freq = (int32)(unit->m_cpstoinc * freqin);
 	float radtoinc = unit->m_radtoinc;
-	LOOP1(inNumSamples,
+
+	int unrolled4 = inNumSamples / 4;
+	int remain4   = unrolled4 - unrolled4 * 4;
```
Comment (danstowell) [inline server/plugins/OscUGens.cpp]: Should this perhaps be `int remain4   = inNumSamples - unrolled4 * 4;` ? You might want to check empirically that an unusual blocksize is working ok.

Comment (crucialfelix) [inline server/plugins/OscUGens.cpp]: good question. ping

Comment (timblechmann) [inline server/plugins/OscUGens.cpp]: fixed

**Comment by timblechmann** [comment]
no need to contribute improvements upstream

**Pull request closed without merging**